File: ios/CustomMethods.m

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(CustomMethods, NSObject)

// Propriedade title que pode ser definida no JavaScript
//RCT_EXPORT_VIEW_PROPERTY(title, NSString)

// Método onPress que pode ser chamado no JavaScript
RCT_EXTERN_METHOD(handlePress:(NSString *)param)

@end

File: ios/CustomMethods.swift
//
//  CustomMethods.swift
//  mistica-react-native
//
//  Created by pablo santos on 08/04/24.
//

import Foundation


@objc(CustomMethods) class CustomMethods: NSObject{
    @objc static func requiresMainQueueSetup() -> Bool {
      return false
    }
    
    @objc public func handlePress(_ param: String){
        print(param)
//        ActionEventModuleManager.emmitter.sendEvent(withName: "onPress", body: [param])
    }
}
